packages/client/src/client/notifications.web.ts

```diff
@@ -139,3 +154,15 @@ export async function registerCreditAcceptedNotifications() {
     console.error(e)
   }
 }
+
+export async function registerCreditCreatedNotifications() {
```

Comment:
I think they should both (FX and Credit) have unregister functions .. with lazy loading and app switching, it will probably show all the wrong notifications